.github/workflows/on_call_do_publish_image.yml

```diff
@@ -0,0 +1,115 @@
+name: CI Release Images
+
+on:
+  workflow_call:
+    inputs:
+      repository:
+        description: 'Name of the repo to download server artifact from'
+        type: string
+      run-id:
```

Comment:
Do we need to explicitly specify this? From the GH page:

```
    # The id of the workflow run where the desired download artifact was uploaded from.
    # If github-token is specified, this is the run that artifacts will be downloaded from.
    # Optional. Default is ${{ github.run_id }}
```

Reply:
no, it's not called directly from the pr/push workflow, due to security reasons. so the initial run id must be passed as input

Reply:
Argh makes sense, I didn't realise we weren't calling it direct.